chains/rari-arbitrum-sepolia-testnet.json

```diff
@@ -11,7 +11,7 @@
     "browserUrl": "https://testnet.explorer.rarichain.org/"
   },
   "id": "1918988905",
-  "name": "RARI Arbitrum Sepolia testnet",
+  "name": "RARI Chain testnet",
```

Comment:
Similar to https://github.com/api3dao/chains/blob/main/chains/gnosis-testnet.json#L15